synthnn/core/musical_extensions.py

```
@@ -231,7 +238,7 @@ def create_modal_connections(self, connection_pattern: str = "nearest_neighbor",
                         weight = weight_scale / (1 + abs(i - j))
                         self.connect(src_id, tgt_id, weight=weight)
                         
-    def generate_chord_progression(self, chords: List[List[float]], 
+    def generate_chord_progression(self, chords: List[object],
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)